src/index.ts

```diff
@@ -338,6 +341,12 @@ export function buildCompanion({
     return component;
   });
 
+  if (projectConfig.appType === AppType.SERVICE && (settings || companion)) {
```

Comment:
Is this temporary? Services will need a companion (and settings should work though there's not an initial use for it).

Reply:
The use cases I had in mind didn't require a companion, so I thought it might be a good idea to add this restriction, as it allowed us to remove it later if we need. Since there are use cases that require it, I removed the restriction. Thanks!